src/stores/HashStateStore.ts

```diff
@@ -30,8 +30,11 @@ export class HashStateStore {
         makeAutoObservable(this)
 
         if (typeof window !== 'undefined') {
```

Comment:
I keep forgetting that we removed SSR completely :D.

They are not used in tests or any similar. So we can safely remove them. The only consideration is for future proofing, in case we will write tests that include the `SharedStoreProvider`. But in that case we can mock the window/global object as well.